cfme/tests/test_rest.py

```diff
@@ -371,31 +684,70 @@ def test_vm_add_lifecycle_event(request, rest_api, vm, from_detail, db):
     ))) == 1, "Could not find the lifecycle event in the database"
 
 
+@pytest.mark.parametrize(
+    "multiple", [False, True],
+    ids=["one_request", "multiple_requests"])
+def test_automation_requests(request, rest_api, automation_requests_data, multiple):
+    """Test adding the automation request
+     Prerequisities:
+        * An appliance with ``/api`` available.
+    Steps:
+        * POST /api/automation_request - (method ``create``) add request
+        * Retrieve list of entities using GET /api/automation_request and find just added request
+    Metadata:
+        test_flag: rest, requests
+    """
+
+    if "automation_requests" not in rest_api.collections:
+        pytest.skip("automation request collection is not implemented in this version")
+
+    if multiple:
+        requests = rest_api.collections.automation_requests.action.create(*automation_requests_data)
+    else:
+        requests = rest_api.collections.automation_requests.action.create(
+            automation_requests_data[0])
+
+    def _finished():
+        flag = False
+        for request in requests:
+            request.reload()
+            if request.status.lower() in {"Error"}:
```

Comment:
I dont think this condition can ever be true - a lowecase string cant be "Error".